modules/apps/commerce/commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/frontend/data/set/provider/CommerceProductDefinitionLinkFDSDataProvider.java

```diff
@@ -105,7 +104,7 @@ public List<ProductLink> getItems(
 							name, "rounded", "lg",
 							cpDefinition.getDefaultImageThumbnailSrc(
 								AccountConstants.ACCOUNT_ENTRY_ID_ADMIN)),
-						HtmlUtil.escape(name), cpDefinitionLink.getPriority(),
+						name, cpDefinitionLink.getPriority(),
```

Comment:
Are all the "users" of these ProductLinks escaping the name?

Reply:
@smottal , the DataProvider is only being used in product relations in the admin view. One "user" as far as I can tell.